packages/playground-examples/copy/ja/3-7/Types and Code Flow/Recursive Type References.ts

```diff
@@ -0,0 +1,39 @@
+//// { compiler: {  }, order: 2 }
+
+// タイプを使うかインターフェースを使うかはそれぞれの機能の制約に依るところがあります。
+// 3.7では、タイプに関する制約のうち、インターフェースにはない制約が削除
+// されました。
+
+// これについての詳細はexample:types-vs-interfacesを参照してください。
+
+// 以前は、定義している型自体の内部では、定義している型を参照することは
+// できませんでした。これはインターフェースには存在しない制約であり、
+// 回避するには少しの作業が必要でした。
+
+// 例えば、これは3.6では実行不可能です:
```

Comment:
IMO: 直訳にしてみました。実行不可能は結構強い言葉に見えるので
```suggestion
// 例えば、以下は3.6では実行できません:
```